src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/HKDF.cs

```diff
@@ -110,10 +115,15 @@ public static byte[] Expand(HashAlgorithmName hashAlgorithmName, byte[] prk, int
         /// <param name="prk">The pseudorandom key of at least <see cref="HashLength"/> bytes (usually the output from Expand step).</param>
         /// <param name="output">The destination buffer to receive the output keying material.</param>
         /// <param name="info">The context and application specific information (can be an empty span).</param>
+        /// <exception cref="ArgumentException"><paramref name="output"/> is larger than the maximum allowed length.</exception>
+        /// <exception cref="ArgumentOutOfRangeException"><paramref name="output"/> is empty.</exception>
```

Comment:
```suggestion
        /// <exception cref="ArgumentException"><paramref name="output"/> is empty.</exception>
```

Since the thrown exception type got updated.